Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -7,121 +7,604 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import Dcrlibwallet
+import Signals
 
-class OverviewViewController: UIViewController {
-    @IBOutlet weak var syncProgressViewContainer: UIView!
+class OverviewViewController: UIViewController{
+    // Heading & balance
+    @IBOutlet weak var logoImage: UIImageView!
+    @IBOutlet weak var overViewLabel: UILabel! {
+        didSet { self.overViewLabel.text = LocalizedStrings.overview }
+    }
+    @IBOutlet weak var balanceLabel: UILabel! {
+        didSet { self.balanceLabel.font = UIFont(name: "Source Sans Pro", size: 40) }
+    }
+    @IBOutlet weak var currentBalanceLabel: UILabel! {
+        didSet {
+            self.currentBalanceLabel.text = LocalizedStrings.currentTotalBalance
+            self.currentBalanceLabel.font = UIFont(name: "Source Sans Pro", size: 14)
+        }
+    }
     
-    @IBOutlet weak var overviewPageContentView: UIView!
-    @IBOutlet weak var fetchingBalanceIndicator: UIImageView!
-    @IBOutlet weak var totalBalanceLabel: UILabel!
-    @IBOutlet weak var recentActivityTableView: UITableView!
+    // Backup phrase section (Top view)
+    @IBOutlet weak var backupWarning: UIView!
     
-    var recentTransactions = [Transaction]()
+    @IBOutlet weak var backupSeedPhrase: UILabel! {
+        didSet{
+            self.backupSeedPhrase.font = UIFont(name: "Source Sans Pro", size: 16)
+            self.backupSeedPhrase.text = LocalizedStrings.backupSeedPhrase
+        }
+    }
+    
+    @IBOutlet weak var backupWarningText: UILabel! {
+        didSet { self.backupWarningText.font = UIFont(name: "Source Sans Pro", size: 14) }
+    }
     
-    override func shouldPerformSegue(withIdentifier identifier: String, sender: Any?) -> Bool {
-        if identifier == "embedSyncProgressVC" && AppDelegate.walletLoader.isSynced {
-            return false
+    // MARK: Transaction history section
+    @IBOutlet weak var recentActivitySection: UIStackView!
+    @IBOutlet weak var recentActivityLabelView: UIView! {
+        didSet { self.recentActivityLabelView.horizontalBorder(borderColor: UIColor(red: 0.24, green: 0.35, blue: 0.45, alpha: 0.5), yPosition: self.recentActivityLabelView.frame.maxY-1, borderHeight: 0.62)}
```

Comment:
Split this into multiple lines. Might be better to place this in the `viewDidLoad` method rather than here.

Reply:
resolved in 9991341